index.html

```diff
@@ -97,7 +118,7 @@ <h3 class="project_heading">Professional Art Printing Data</h3>
         <button class="see_project button">See Project</button>
       </div>
 
-      <section class="work_card">
+      <section class="work_card card">
```

Comment:
- [ ] Kindly implement a `::hover` pseudoclass here, so when the user hovers the element, the opacity is gone and a button appears to go to the project.  You could use `display-none` for the language tags, and a `display-block` for the button on hover.  That way, your hovers on the desktop will look beautiful and functional 💪